test-base.cfg

```diff
@@ -215,7 +221,7 @@ initialization =
         shell=True, stdout=subprocess.PIPE).stdout.read().strip()
     if '--absolute' in sys.argv: pkgdir = os.path.abspath(pkgdir) ; sys.argv.remove('--absolute')
     if len(sys.argv) < 2: sys.argv.extend([pkgdir])
-    sys.argv.extend(${pycodestyle:args})
+    sys.argv.extend(${pycodestyle-ci-run:args})
```

Comment:
In my emacs configuration I run `./bin/pycodestyle --absolute`, expecting the source of the current package to be scanned. Do I need to switch to `bin/pycodestyle-ci-run` then?
I really don't want all dependencies in `./src` do be scanned, nor do I want `parts/omlette` to be scanned.

And:
```
$ ./bin/buildout
warning: no files found matching '*.txt'
warning: no previously-included files matching '._*' found anywhere in distribution
warning: no previously-included files matching '*.mo' found anywhere in distribution
While:
  Installing.
  Getting section pycodestyle-ci-run.
  Initializing section pycodestyle-ci-run.
  Getting option pycodestyle-ci-run:initialization.
Error: Referenced option does not exist: pycodestyle-ci-run args
```